### PR TITLE
Dg bugfixes

### PR DESCRIPTION
**Pull request by sueann-chua**
Fixed typos
Reorder User Stories

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -326,29 +326,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+| ` * * `  | user preparing interviews | be reminded of what interviews I have the next day | keep track of them |
+| ` * * `  | user preparing interviews | be able to reschedule the interview session for a candidate | so that I can keep up with changes coming from the candidate's end |
+| ` * * `  | user preparing interviews | be able to delete the interview sessions on a particular date | |
 | ` * * `  | user preparing interviews | search what interviews I have for a particular date | better prepare for that day |
-| ` * * `  | user       | see all the candidates scheduled for interview sessions for a particular job posting | |
-| `* * *`  | user       | be able to search for a particular candidate to see his/her upcoming sessions | |
+| `  *  `  | user preparing interviews | add a co-interviewer to an interview | I know who I will be interviewing the candidate with |
 | `  *  `  | user       | be able to password lock the application to prevent unauthorised access | |
 | `  *  `  | user       | encrypt the save file | prevent my data from being easily stolen |
```
Comment (wanyu-l) [inline docs/DeveloperGuide.md:351]: I think the spaces between asterisk and ` need to be removed too for the DG webpage to display correctly

**Comment by seaweediman** [comment]
Probably cause you merged my PR first, but now the DG has conflicts.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (sueann-chua) [reply docs/DeveloperGuide.md]: resolved

**Review by wanyu-l — APPROVED** [review]
LGTM.

**Pull request merged**